### PR TITLE
Config service

### PR DESCRIPTION
**Pull request by ItsAlxl**

### CODE REVIEW

File: ConfigService/src/main/java/com/revature/ConfigServlet.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+import org.kohsuke.github.GHOrganization;
+import org.kohsuke.github.GHOrganization.Permission;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GHUser;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ConfigServlet extends HttpServlet {
+    final String ORG_NAME = "revaturelabs";
```
Comment (Bensonre) [inline ConfigService/src/main/java/com/revature/ConfigServlet.java:27]: Could we set these with a config file? That way it is easy for Revature to change without changing source code.

File: ConfigService/src/test/java/com/revature/AppTest.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        assertEquals(projNameExpected, servlet1.projName);
+        assertEquals(usingJenkinsExpected, servlet1.usingJenkins);
+        assertEquals(debugModeExpected, github.debugMode);
+    }
+    
+    /**
+     * Rigorous Test :-)
+     */
+    @Test
+    public void shouldAnswerWithTrue()
```
Comment (Bensonre) [inline ConfigService/src/test/java/com/revature/AppTest.java:48]: Remove this test

**Review by Bensonre — APPROVED** [review]

File: ConfigService/kube/autoci-config-service.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: autoci-config-svc
+  namespace: autoci
+spec:
+  type: NodePort
+  selector:
```
Comment (re-blank) [inline ConfigService/kube/autoci-config-service.yaml:8]: Might want to change to LoadBalancer here.

**Review by re-blank — APPROVED** [review]
No major problems.

File: ConfigService/BuildAndPush_ConfigService.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+mvn clean package
+echo "Enter Docker Hub username"
+read username
+docker build -t $username/autoci-config:latest $(dirname $0)
+docker push $username/autoci-config:latest
```
Comment (re-blank) [inline ConfigService/BuildAndPush_ConfigService.sh:5]: Should migrate to ECR if we have time.

**Pull request merged**